### ISSUE TITLE
Crashed a lot

### ISSUE THREAD

**Issue by lexrus** [ytakzk/Fusuma#33]
I have no idea what happened.

<img width="959" alt="screen shot 2016-04-28 at 1 49 04 pm" src="https://cloud.githubusercontent.com/assets/219689/14876525/504fa58c-0d48-11e6-93c1-341a082030f8.png">

**Comment by ytakzk** [comment]
@lexrus 

Don't you call the view controller asynchronously?
Could you show me more logs if possible?

**Comment by lexrus** [comment]
I call it in a IBAction.

``` swift
@IBAction func pickAvatar(sender: AnyObject) {
        let fusuma = FusumaViewController()
        fusumaTintColor = UIColor.hex("0175C8")
        fusumaBackgroundColor = UIColor.hex("1D2227")
        fusuma.defaultMode = .Camera
        fusuma.modeOrder = .CameraFirst
        fusuma.delegate = self
        presentViewController(fusuma, animated: true, completion: nil)
}
```

**Comment by ytakzk** [comment]
@lexrus Sorry I cannot reproduce it..
Do you have more information? It would be super helpful for me to fix the issue.

**Comment by ytakzk** [comment]
@lexrus Maybe this has been solved on v0.4.1.
Thanks!
